File: code/obsolete/sinema_salonu.m
clear all; close all; clc;
n = 1e5;
together = 0;
for i=1:n
    seat = randi(100, 1, 2);
    while (seat(1) == seat(2))
        seat(2) = randi(100);
    end
    if (abs(seat(1)-seat(2)) == 1) && (ceil(seat(1)/10) == ceil(seat(2)/10))
        together = together+1;
    end
end
fprintf('Olasılık (analitik) = %.5f\n', 1/55);
fprintf('Olasılık (nümerik) = %.5f\n', together/n);

File: code/obsolete/sinema_salonu.py
from numpy.random import randint
import numpy as np
n = 50000
m = 0
xy = randint(1,101,(n,2))
for i in range(n):
    while xy[i,0] == xy[i,1]:
        xy[i,1] = randint(1,101)
    if (abs(xy[i,0]-xy[i,1]) == 1) and (np.ceil(xy[i,0]/10) == np.ceil(xy[i,1]/10)):
        m += 1
print('Yan yana oturma olasılığı (analitik) = %.5f' %(1/55))
print('Yan yana oturma olasılığı (nümerik) = %.5f' %(m/n))